packages/react-ui/components/Modal/__stories__/Modal.stories.tsx

```diff
@@ -767,3 +751,31 @@ export const AlignCenterAndNoClose = () => (
   </Modal>
 );
 AlignCenterAndNoClose.story = { parameters: { creevey: { captureElement: null } } };
+
+const Header = () => <Modal.Header>Header</Modal.Header>;
+const Body = () => (
+  <Modal.Body>
+    {new Array(200).fill('Use rxjs operators with react hooks.').map((item, index) => (
+      <p key={index}>{item}</p>
+    ))}
+  </Modal.Body>
+)
+const Footer = () => <Modal.Footer>Footer</Modal.Footer>;
```

Comment:
Для `<Modal.Footer>` можно добавить проп `panel`. Так мы будем тестировать сразу все нужные состояния.

```suggestion
const Footer = () => <Modal.Footer panel>Footer</Modal.Footer>;
```